ui/app/AppLayouts/ActivityCenter/controls/NotificationCard.qml

```diff
@@ -0,0 +1,322 @@
+        property real factorXL:   1.20
+        property real factorXXL:  1.30
+
+        // Dot size used by header spacing.
+        readonly property int readUnreadBadgeSize: 8
+
+        // Updates card background according to selection state.
+        function updateCardBackgroundColor() {
+            if (root.selected) {
+                bg.color = Theme.palette.baseColor5
+            } else {
+                bg.color = Theme.palette.transparent
+            }
+        }
+
+        // Returns the avatar scaling factor for a given font size enum value.
+        function avatarFactorForFontSize(fs) {
+            switch (fs) {
+            case Theme.FontSize.FontSizeXS:  return d.factorXS;
+            case Theme.FontSize.FontSizeS:   return d.factorS;
+            case Theme.FontSize.FontSizeM:   return d.factorM;
+            case Theme.FontSize.FontSizeL:   return d.factorL;
+            case Theme.FontSize.FontSizeXL:  return d.factorXL;
+            case Theme.FontSize.FontSizeXXL: return d.factorXXL;
+            default:                         return 1.0;  // Safe fallback
+            }
+        }
+    }
+
+    // Keep background consistent when selection changes.
+    onSelectedChanged: d.updateCardBackgroundColor()
+
+    // Card background and unread indicator.
+    background: Rectangle {
+        id: bg
+        anchors.fill: parent
+        radius: 8
+        color: root.selected ? Theme.palette.baseColor5 : Theme.palette.transparent
+
+         // Unread indicator dot (top-right).
+        Rectangle {
+            anchors.right: parent.right
+            anchors.top: parent.top
+            anchors.margins: Theme.halfPadding
+            width: Theme.halfPadding
+            height: width
+            radius: width / 2
+            color: root.unreadDotColor
+            visible: root.unread
+        }
+    }
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // Content - Main layout: avatar + content column.
+    // ──────────────────────────────────────────────────────────────────────────
+    contentItem: RowLayout {
+        spacing: root.spacing
+
+        // Avatar block (non-clickable here; card handles clicks).
+        NotificationAvatar {
+            Layout.alignment: Qt.AlignTop
+            Layout.leftMargin: Theme.halfPadding
+
+            // Scale avatar with current font size factor
+            density: d.avatarFactorForFontSize(Theme.currentFontSize)
+
+            avatarSource: root.avatarSource
+            badgeIconName: root.badgeIconName
+            circular: root.isCircularAvatar
+            isAvatarClickable: false
+            isBadgeClickable: false
+        }
+
+        // Main content area
+        ColumnLayout {
+            spacing: Theme.smallPadding / 2
+            Layout.fillWidth: true
+            Layout.rightMargin: Theme.halfPadding
+
+            // Header row: title + chat key + contact/trust badges.
+            NotificationHeaderRow {
```

Comment:
Not a must, but I would consider keeping such helper sub-components simpler and use e.g. `RowLayout` as a base instead of full `Control`. Especially when the component itself has `Row` in the name, stating clearly it's a row in fact. It would make the final delegate bit lighter.